controllers/space/space_controller.go

```diff
@@ -197,16 +196,132 @@ func (r *Reconciler) ensureNSTemplateSet(logger logr.Logger, space *toolchainv1a
 	}, tmplTier); err != nil {
 		return norequeue, r.setStatusProvisioningFailed(logger, space, err)
 	}
+
+	spaceBindings, err := r.listSpaceBindings(space)
+	if err != nil {
+		logger.Error(err, "failed to list space bindings")
+	}
+	nsTmplSetReady, duration, err := r.manageNSTemplateSet(logger, space, memberCluster, spaceBindings, tmplTier)
```

Comment:
it's just a detail - the `listSpaceBindings` method could have been called directly from `manageNSTemplateSet` as the retrieved spaceBindings are used only there

Reply:
yeah , good point!

changed in [6b9e585](https://github.com/codeready-toolchain/host-operator/pull/733/commits/6b9e5853271c52ff603d67960dbfd63ba4f10904)